styleguide/source/_patterns/03-organisms/ipe-actions.twig

```diff
@@ -0,0 +1,39 @@
+<div class="ipe-actions-block ipe-actions" data-block-action-id="" data-block-edit-id="">
+  <h5>Block: Title of Block</h5>
+  <ul class="ipe-action-list">
+    <li data-action-id="remove">
+      <a>
+        <span class="ipe-icon ipe-icon-remove"></span>
+      </a>
+    </li>
+    <li data-action-id="up">
+      <a>
+        <span class="ipe-icon ipe-icon-up"></span>
+      </a>
+    </li>
+    <li data-action-id="down">
+      <a>
```

Comment:
can the classes go into the a not striped out the spans?
a tags need href to be valid

Reply:
This code comes from Drupal and there doesn't seem to be a theme suggestion available to hook into to modify it.